base/src/main/java/com/smartdevicelink/managers/screen/BaseTextAndGraphicManager.java

```diff
@@ -409,7 +410,7 @@ Show assembleShowText(Show show){
 			return show;
 		}
 
-		int numberOfLines = getNumberOfLines();
+		int numberOfLines = (defaultMainWindowCapability != null && defaultMainWindowCapability.getTextFields() != null) ? ManagerUtility.WindowCapabilityUtility.getMaxNumberOfMainFieldLines(defaultMainWindowCapability) : 4;
```

Comment:
Does it max sense to just alter the utility method call to return 4 by default instead of this long inline if statement? This is not a hard requirement and would be interested to know how iOS handles it.

Reply:
We kind of abstracted the logic in the helper methods in `ManagerUtility` to just return the actually supported things. And we kept the implementation for the rule "if null, then all enabled" directly in the manager's implementation. Of course, that is not a hard rule. We can apply the suggestion if you think that makes more sense. I just did it this way for alignment purposes as iOS implementation also does it similarly: 
https://github.com/smartdevicelink/sdl_ios/blob/d4bd0eed98cb1e872da5bcd4a62b04d81a87f815/SmartDeviceLink/SDLTextAndGraphicManager.m#L315:

Reply:
If that's how iOS does it then staying aligned is more important right now. We can always address a better method for both in the future.